ntripclient.c

```diff
@@ -845,14 +845,15 @@ int main(int argc, char **argv)
       // this block of code is very long and difficult to read.  I'll attempt
       // to summarize here.
       //
-      // Lines 747 - 1080 handle UDP mode
-      // Lines 1082 - 1433 handle RTSP mode
-      // Lines 1435 - 1822 handle all the other modes
-      // 
```

Comment:
@rabad-matternet I removed the line numbers so we don't have to keep maintaining them. I changed it to say there are three major sections to look out for.

Reply:
I agree that line numbers are problematic to maintain.  How about putting a comment that can be searched at each section and then state what that searchable comment is here?  

The motivation for the line numbers (and now a searchable comment) is that this function is to big to see what's going on.  So having a way to easily navigate would be nice.

Reply:
Done. Great suggestion.